packages/material-ui-types/index.d.ts

```diff
@@ -1,3 +1,5 @@
+import React = require('react');
```

Comment:
```suggestion
import * as React from 'react';
```

Reply:
This would results in a ci failure.

Reply:
That's weird. We import it like that in every other declaration.

Reply:
See https://circleci.com/gh/mui-org/material-ui/101111?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
You didn't use the wildcard import in that build though.